### ISSUE TITLE
[Accessibility] [Accordion] Single Item Expansion aria-expanded state not read properly in VO + Safari

### ISSUE THREAD

**Issue by CameronSharp1980** [adobe/aem-core-wcm-components#1466]
## Bug Report

**Current Behavior**
When navigating the accordion component with Voiceover + Safari and activating the accordion button with VO Key + Space, the expanded / collapsed state of the accordion is not read properly, if at all. 

**Expected behavior/code**
The aria-expanded state of the accordion should be read out when accordion is toggled. (Safari + VO in this instance)

**Environment**
- AEM version and patch level: 6.5.6.0 & AEM as a cloud service
- Core Components version: 2.15.0
- JRE version: Java(TM) SE Runtime Environment (build 1.8.0_241-b07)

**Possible Solution**
I have had some success with similar issues before in VO + Safari by changing the order that DOM visibility states and aria attributes are toggled and/or by using a setTimeout to ensure that aria states are toggled before or after DOM visibility changes are complete as needed. (As I recall, even a setTimeout of 0ms forces the encapsulated code to be parsed after everything outside.)

**Additional context / Screenshots**
All accordion states seem to be affected by this and are likely at least partially covered by webkit bug reports such as:
 - https://bugs.webkit.org/show_bug.cgi?id=92066
 - https://bugs.webkit.org/show_bug.cgi?id=210934
 - https://bugs.webkit.org/show_bug.cgi?id=212471 

-However-
The Single Item Expansion option seems to only partially overlap with this issue as this pattern does work from w3:
 - https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html
